app/helpers/index.js

```diff
@@ -0,0 +1,22 @@
+import { createElement as h } from "react";
+export { tsToDate } from "./dateFormat";
+export { addSpacingAroundText, restrictToStdDecimalNumber } from "./strings";
+
+export const kidCheck = C => {
```

Comment:
Can you please add comments for the less than clear helper funcs please?

Reply:
added